### ISSUE TITLE
Modularization

### ISSUE THREAD

**Issue by markborkum** [Megultron/Ruby-PeakList-Library#2]
Currently, there is one source code file `peakListParser.rb` that exports five separate `class`es. The names for each `class` are exported to the global namespace, and may conflict with other imports.

Consider introducing a new top-level `module`:

```ruby
module Bruker
  module XML
    class Document
      class << self
        def parse(object)
          # TODO
        end
      end
    end
  end

  class PeakList
  end

  class Peak
  end
end
```

**Comment by Megultron** [comment]
Will do. Would it be more intuitive to define the parser and data structures in separate modules, or would it be cleaner to keep them all bundled together?

**Comment by markborkum** [comment]
Either works. The Ruby interpreter will create the necessary `class`es and `module`s in the order that they are `require`d.

For the moment, I would suggest keeping everything in one file, as part of a top-level `module`.

**Comment by Megultron** [comment]
Done. All the PeakList classes should be under the new module 'PeakListParse'.

**Comment by Megultron** [comment]
Fixed with commit ae9f286dabefc615fc20b4010dcd1d49986e3372
